src/node.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+export default class Node {
+  constructor(data){
+    this.data = data
+    this.next = undefined 
```

Comment:
undefined variable declarations can be very dangerous in javascript, heres a link to a stack overflow response that explains it fairly well: http://stackoverflow.com/questions/2235622/can-i-set-variables-to-undefined-or-pass-undefined-as-an-argument